ThingIFSDK/ThingIFSDK/IoTOperations/IoTRequestOperation.swift

```diff
@@ -33,7 +33,13 @@ func buildDefaultRequest(method : HTTPMethod,urlString: String,requestHeaderDict
     kiiVerboseLog("Request Method: \(method)")
     kiiVerboseLog("Request Header: \(requestHeaderDict)")
 
-    let defaultRequest = DefaultRequest(method: method, urlString: urlString, requestHeaderDict: requestHeaderDict, requestBodyData: requestBodyData, responseBodySerializer: { (responseBodyData) -> NSDictionary? in
+    // Add X-Kii-SDK header.
+    var modifiedHeaderDict:Dictionary<String, String> = requestHeaderDict;
+    let b:NSBundle? = NSBundle.allFrameworks().filter{$0.bundleIdentifier == "Kii-Corporation.ThingIFSDK"}.first
```

Comment:
IMO, this is not efficient. this will be called each time on request generation.
I think it will be better to store the header string on a singleton read-only property. Since the value will be the same in every runtime.